contracts/ServiceRegistryL2.sol

```diff
@@ -1,9 +1,8 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.15;
+pragma solidity ^0.8.19;
 
 import "./GenericRegistry.sol";
 import "./interfaces/IMultisig.sol";
-import "./interfaces/IRegistry.sol";
```

Comment:
As we discussed this is not relevant, can you confirm that IService is not relevant as well?